src/output.py

```diff
@@ -94,33 +92,36 @@ def getEditorAndPath():
     return 'vim', 'vim'
 
 
-def getEditFileCommand(filePath, lineNum):
```

Comment:
ah i see why this had to go -- yeah you didnt have global context when composing these mini-parts.